projects/cairographics.org/pycairo/package.yml

```diff
@@ -0,0 +1,26 @@
+distributable:
+  url: https://github.com/pygobject/pycairo/releases/download/v{{version}}/pycairo-{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: pygobject/pycairo
+dependencies:
+  cairographics.org: '*'
+runtime:
+  env:
+    PYTHONPATH: "{{prefix}}/lib/python3.11/site-packages:$PYTHONPATH"
+build:
+  dependencies:
+    freedesktop.org/pkg-config: '*'
+    python.org: ^3.11
```

Comment:
If you're going to lock the directory via `PYTHONPATH`, then I think you probably need to lock the version as well, to `~3.11`. Possibly that could instead be worked around with symlinks? `ln -s python{{version.marketing}} python{{version.major}}`?

Reply:
I did with {{deps.python.org.version.marketing}}
So will it be ok?